validation-test/stdlib/Set.swift

```diff
@@ -1141,10 +1128,21 @@ SetTestSuite.test("deleteChainCollision2") {
   expectTrue(s.contains(k6_0))
 }
 
-SetTestSuite.test("deleteChainCollisionRandomized") {
-  let timeNow = CUnsignedInt(time(nil))
-  print("time is \(timeNow)")
-  srandom(timeNow)
+func product<C1 : Collection, C2 : Collection>(
+  _ c1: C1, _ c2: C2
+) -> [(C1.Iterator.Element, C2.Iterator.Element)] {
+  var result: [(C1.Iterator.Element, C2.Iterator.Element)] = []
+  for e1 in c1 {
+    for e2 in c2 {
+      result.append((e1, e2))
+    }
+  }
+  return result
+}
```

Comment:
This is copied from [validation-test/stdlib/Dictionary.swift](https://github.com/apple/swift/blob/8c6df3647b3eec0ee526051e412bebb2c83b29d0/validation-test/stdlib/Dictionary.swift#L1505-L1519)